src/app/components/menubar/menubar.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { BrowserWindow } from 'electron';
+
+@Component({
+  selector: 'app-menubar',
+  templateUrl: './menubar.component.html',
+  styleUrls: ['./menubar.component.scss']
+})
+export class MenubarComponent {
+  isFullscreen: boolean;
+  window;
+  fullscreenIcon: string;
+
+  constructor() {
+    this.window =  require('electron').remote.getCurrentWindow();
```

Comment:
since you're using require('electron') in a couple spots, it might be best to extract that to a common variable (i.e., var electron = require('electron'))